src/app/_components/CustomLink.tsx

```diff
@@ -25,13 +27,11 @@ export function CustomLink({
     )
   }
 
-  const rel = href.startsWith('mailto:') ? undefined : 'noopener noreferrer'
-
   return (
     <a
-      href={href}
       rel={rel}
-      aria-label="Opens external link"
+      title={title || 'External link'}
```

Comment:
As I understand, the role of the `title` is to clarify the purpose of the link, like where the link will send the user who clicks on that link? [Example code from W3C ](https://www.w3.org/TR/WCAG20-TECHS/H33.html)

```
<a href="http://example.com/WORLD/africa/kenya.elephants.ap/index.html" 
   title="Read more about failed elephant evacuation">
   Evacuation Crumbles Under Jumbo load
</a>
```
So not sure if "External link" is helpful / servers the purpose?

Reply:
Also seems redundant when we have aria-label?

![Screenshot 2024-11-07 at 13 17 58](https://github.com/user-attachments/assets/cfff3169-b406-417e-9c05-72b71cbf814c)

Reply:
The HTML `title` and `aria-label` attributes serve different purposes and it's fine to use both:

- `title` creates a visual tooltip that appears when users hover over the link with their mouse - this helps sighted users understand where the link goes before clicking
- `aria-label` is specifically for screen readers and doesn't have any visual effect - this helps users who can't see the tooltip get the same contextual information.

(For example, for the copy to clipboard link, and social icons, the `title` attribute is quite helpful.`)
But you are right that we shouldn't fallback to `External link` - We can leave it empty if it's not provided.

Reply:
Nice a11y refresher :)